torch_glow/src/PyTorchModelLoader.h

```diff
@@ -46,12 +46,19 @@ class PyTorchModelLoader {
   /// Mapping from PyTorch Values to Glow NodeValues created during loading.
   std::unordered_map<const torch::jit::Value *, glow::NodeValue> valueMap_;
```

Comment:
What if the same input used more than one time, does it mean Value* will be the same? May by Value::uniqueName is a better key?

Reply:
> What if the same input used more than one time
Unless I misunderstand your question, it should be fine. glow::NodeValue can also be used multiple times.